ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-master-presubmits.yaml

```diff
@@ -120,108 +120,108 @@ presubmits:
     always_run: true
     branches:
     - master
-    context: ci/prow/e2e-aws-upgrade
+    context: ci/prow/e2e-aws-scaleup-rhel7
```

Comment:
just to clarify, you made some tweaks to rhel scaleup here and also reordered the upgrade & scaleup in this file?

Reply:
@kikisdeliveryservice these files are generated by prowgen. I'm guessing it tries to keep them in alphabetical order, which unfortunately is making this harder to review. As best as I can tell there were no unintended changes to the upgrade job.

Reply:
ahhh thank you!! did not realize!